pkgs/applications/science/misc/netlogo/default.nix

```diff
@@ -0,0 +1,43 @@
+{ jre, stdenv, fetchurl, makeWrapper }:
+
+
+stdenv.mkDerivation rec {
+  name = "NetLogo-${version}";
+  version = "6.0.4";
+
+
+  src = fetchurl {
+    url = "https://ccl.northwestern.edu/netlogo/6.0.4/NetLogo-6.0.4-64.tgz";
+    sha256 = "0dcd9df4dfb218826a74f9df42163fa588908a1dfe58864106936f8dfb76acec";
+  };
+
+
+  buildInputs = [ makeWrapper ];
```

Comment:
I think this should be a `nativeBuildInputs`.

Reply:
Yeah, I wasn't sure about that. AFAIK there are examples of `makeWrapper` being in `buildInputs` as well in nixpkgs. Nevertheless, I'll change it!